### PR TITLE
ENH: add support for multiple operations in a request.

### PR DESCRIPTION
**Pull request by cournape**
@johntyree

### CODE REVIEW

File: simplesat/transaction.py [hunk 1]
```diff
@@ -1,23 +1,26 @@
 from collections import OrderedDict
 
+from attr import attr, attributes
```
Comment (johntyree) [inline simplesat/transaction.py]: Where does this come from? Is it a package that must be installed?

**Comment by johntyree** [comment]
Code looks reasonable. One question about an import.

File: simplesat/transaction.py (hunk 1 above)
Comment (cournape) [inline simplesat/transaction.py]: Yes, it is attrs: https://pypi.python.org/pypi/attrs

Comment (cournape) [inline simplesat/transaction.py]: Let me add it as a dependency, I thought we already had it

**Pull request merged**